source/help/apps/desktop-changelog.rst

```diff
@@ -4,9 +4,13 @@ Desktop Application Changelog
 Release v4.5
 ----------------------------
 
-**Release date:** June 16th, 2020
+**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/tag/v4.5.1>`_
 
-**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/tag/v4.5.0>`_
+- **v4.5.1, released 2020-07-13**
+ - Mattermost v4.5.1 contains a high level security fix. `Upgrading <http://docs.mattermost.com/administration/upgrade.html>`__ is highly recommended. Details will be posted on our `security updates page <https://about.mattermost.com/security-updates/>`__ 30 days after release as per the `Mattermost Responsible Disclosure Policy <https://www.mattermost.org/responsible-disclosure-policy/>`__.
+    - Known issue: If both “Show icon in the notification area" and "Start app on login" are enabled, the Linux app starts as a blank screen. `MM-26832 <https://mattermost.atlassian.net/browse/MM-26832>`_
```

Comment:
```suggestion
    - Known issue: If both “Show icon in the notification area" and "Start app on login" are enabled, the Linux app starts as a blank screen. `MM-26832 <https://mattermost.atlassian.net/browse/MM-26832>`_ clicking on the tray icon and selecting one of the servers will discard this blank page and let the app recover from this state.
```
this might need better wording, but adding a workaround for those who suffer the bug might be useful